src/microbe_stage/organelle_components/CiliaComponent.cs

```diff
@@ -1,6 +1,9 @@
 ﻿using System;
+using System.Diagnostics.CodeAnalysis;
 using Godot;
 
+[SuppressMessage("Design", "CA1001:Types that own disposable fields should be disposable",
```

Comment:
Oh and also now this suppression should be removed and the variables ignored similarly to all the other code, which I did in a big PR yesterday).

Reply:
And done.